models/liquidifty/bnb/liquidifty_bnb_trades.sql

```diff
@@ -0,0 +1,212 @@
+        case
+            when currency = '0x0000000000000000000000000000000000000000' then '{{ wbnb_address }}'
+            else currency
+        end as currency_contract,
+        collection as nft_contract_address,
+        contract_address  as project_contract_address,
+        evt_tx_hash as tx_hash,
+        evt_block_number as block_number,
+        concat(cast(evt_block_number as varchar(5)), '-', cast(evt_index as varchar(5))) as unique_trade_id,
+        case
+            when currency = '0x0000000000000000000000000000000000000000' then 'native'
+            else 'erc20'
+        end as currency_token_standard,
+        '1' as orderType
+    from (
+        select * from {{ source('liquidifty_bnb', 'MarketplaceV2_evt_Buy') }}
+        union all
+        select * from {{ source('liquidifty_bnb', 'MarketplaceV2_1_evt_Buy') }}
+        union all
+        select * from {{ source('liquidifty_bnb', 'MarketplaceV2_5_evt_Buy') }}
+    ) a
+),
+stack as (
+    select
+        cast(null as varchar(5)) as version,
+        evt_block_time as block_time,
+        cast(null as varchar(5)) as token_id,
+        'erc721' as token_standard,
+        'Bundle Item Trade' as trade_type,
+        amount as number_of_items,
+        'Buy' as trade_category,
+        owner as seller,
+        buyer,
+        price as amount_raw,
+        case
+            when currency = '0x0000000000000000000000000000000000000000' then '{{ wbnb_address }}'
+            else currency
+        end as currency_contract,
+        collection as nft_contract_address,
+        contract_address  as project_contract_address,
+        evt_tx_hash as tx_hash,
+        evt_block_number as block_number,
+        concat(cast(evt_block_number as varchar(5)), '-', cast(evt_index as varchar(5))) as unique_trade_id,
+        case
+            when currency = '0x0000000000000000000000000000000000000000' then 'native'
+            else 'erc20'
+        end as currency_token_standard,
+        '1' as orderType
+    from {{ source('liquidifty_bnb', 'MultipleSell_evt_Buy') }}
+),
+v3 as (
+    select
+        'v3' as version,
+        call_block_time as block_time,
+        get_json_object(nft, '$.id') as token_id,
+        case
+            when get_json_object(nft, '$.assetType') = '0' then 'native'
+            when get_json_object(nft, '$.assetType') = '1' then 'erc20'
+            when get_json_object(nft, '$.assetType') = '2' then 'erc721'
+            when get_json_object(nft, '$.assetType') = '3' then 'erc1155'
+        end as token_standard,
+        case
+            when cardinality(orders) > 1 then 'Bundle Trade'
+            else 'Single Item Trade'
+        end as trade_type,
+        get_json_object(nft, '$.amount') as number_of_items,
+        case
+            when orderType = '0' then 'Swap'
+            when orderType = '1' then 'Buy'
+            when orderType = '2' then 'Sell'
+        end as trade_category,
+        get_json_object(order, '$.signer') as seller,
```

Comment:
This might not always hold when the order is a buy order? (offer accepted)
We can get the seller from the `Buy` event.